controllers/submariner/globalnet_resources.go

```diff
@@ -91,6 +92,7 @@ func newGlobalnetDaemonSet(cr *v1alpha1.Submariner, name string) *appsv1.DaemonS
 								{Name: "SUBMARINER_NAMESPACE", Value: cr.Spec.Namespace},
 								{Name: "SUBMARINER_CLUSTERID", Value: cr.Spec.ClusterID},
 								{Name: "SUBMARINER_EXCLUDENS", Value: "submariner-operator,kube-system,operators,openshift-monitoring,openshift-dns"},
+								{Name: "SUBMARINER_GNMETRICSPORT", Value: constants.MetricsGNPort},
```

Comment:
Since the env var is only consumed inside that specific container, I think we can use a simple consistent name.

```suggestion
								{Name: "SUBMARINER_METRICS_PORT", Value: constants.MetricsGNPort},
```

Reply:
This needs to match with what is defined in `submariner` repo. Changing this env here will also require change over there.

Reply:
Seems logical to me to do, unless you think theres a problem with my suggestion.

Also I don't see a change adding this to Submariner, shouldn't this PR exist there first (and current PR should depend on it)?

Reply:
This sets the environment variable up, we need that before Submariner can consume it. Submariner will continue working as-is with this change, but once the environment variable is configured in the template, we can change Submariner to use it.